x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/components/delete_button.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC } from 'react';
+import { EuiButton } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { ml } from '../../../../services/ml_api_service';
+import { getToastNotifications } from '../../../../util/dependency_cache';
+import { JOB_MAP_NODE_TYPES } from '../../../../../../common/constants/data_frame_analytics';
+
+interface Props {
+  id: string;
+  type: JOB_MAP_NODE_TYPES;
+}
+
+export const DeleteButton: FC<Props> = ({ id, type }) => {
+  const toastNotifications = getToastNotifications();
+
+  const onDelete = async () => {
+    try {
+      // if (isDataFrameAnalyticsFailed(d.stats.state)) {
+      //   await ml.dataFrameAnalytics.stopDataFrameAnalytics(d.config.id, true);
+      // }
```

Comment:
commented code

Reply:
The delete functionality will be added in in a follow up.

Reply:
Makes sense trying to keep the PR concise! But I believe we should not commit commented code. You can always stash it locally. [This article](https://kentcdodds.com/blog/please-dont-commit-commented-out-code) provides an explanation of why, and there are plenty of others